src/actions.js

```diff
@@ -93,6 +93,55 @@ module.exports = {
 			},
 		}
 
+		actions.set_active_app = {
+			name: 'Launch App',
+			options: [
+				{
+					type: 'dropdown',
+					label: 'Type',
+					id: 'type',
+					choices: [
+						{ id: 'activity', label: 'Android Activity' },
+						{ id: 'url', label: 'URL' },
+						{ id: 'manifest', label: 'Manifest' },
+						{ id: 'auid', label: 'Application ID' },
+					],
+				},
+				{
+					type: 'textinput',
+					label: 'Value',
+					id: 'app_value',
+					default: '',
+					required: true,
+				},
+			],
+			callback: async function (action) {
+				let opt = action.options
+				const value = encodeURI(opt.app_value.trim())
+				let params = {
+					uri: opt.type === 'activity' ? '' : 'localapp://webappruntime?',
+				}
+				if (opt.type === 'url') {
+					params.uri += 'url=' + value
+				} else if (opt.type === 'manifest') {
+					params.uri += 'manifest_url=' + value
```

Comment:
Seems to be this from the docs:
```suggestion
					params.uri += 'manifest=' + value
```

Reply:
Good point. I didn't notice it. Thanks.